src/controller.js

```diff
@@ -47,6 +50,28 @@ class ChatbotCtrl {
 
     this.$scope.$on('ovh-chatbot:open', () => this.open());
     this.$scope.$on('ovh-chatbot:opened', () => this.focusInput());
+
+    const config = this.config || {};
+    if (isString(this.url)) {
```

Comment:
Maybe a console message to tell the users that their url is in the wrong format would be nice :)

Reply:
You mean during compilation? Because that parameter is only passed once by the developer who integrates the chatbot.